src/App.css

```diff
@@ -76,20 +77,18 @@ ul {
   display: flex;
   align-items: center;
   justify-content: center;
+  margin-left: 40%;
 }
 
```

Comment:
- [ ] I've noticed that the header design doesn't quite match the original concept. Can you adjustments to bring it closer to the initial vision of it :upside_down_face: 
Your design 
![Screenshot from 2023-09-01 18-31-10](https://github.com/faisalakb/BookStore/assets/88894525/df7d9ffe-08aa-4390-973e-31c429b01dc8)

The zeplin design
![Screenshot from 2023-09-01 18-29-42](https://github.com/faisalakb/BookStore/assets/88894525/90754daf-1efa-4cd2-8e28-519b5fe4ce33)